support/client/lib/vwf/model/mil-sym.js

```diff
@@ -481,6 +535,14 @@ define( [ "module",
                     case "mobility":
                     case "taskForce":
                     case "installation":
+                    case "x":
+                    case "y":
+                    case "width":
+                    case "height":
+                    case "controlPts":
+                    case "visible":
+                    case "listening":
+                    case "symbolType":
```

Comment:
For the Konva properties (`x`, `y`, `width`, `height`, `visible`, `listening`?), if the intent is to monitor but not own them, you should probably not handle them in `gettingProperty`. This driver does copy the `x`, `y`, `width`, and `height` values and could answer for those properties. But it's better to fall through and let the Konva driver handle them since it owns them.

Reply:
The missionGfx class itself doesn't own them.  I'll remove them.